databases/postgresql.md

```diff
@@ -0,0 +1,259 @@
+# PostgreSQL
+
+[PostgreSQL](http://www.postgresql.org/) is an object-relational database management system (ORDBMS).
+
+The connection between PostgreSQL and Ruby happens via the [`pg`](https://rubygems.org/gems/pg) gem.
+
+## Installation
+
+[Install PostgreSQL](http://www.postgresql.org/download/)
+
+```
+gem install pg
```

Comment:
Does this need some description? something like "After the DB in installed on the machine, install the driver" etc.?